Sources/Sentry/include/SentryNSDataTracker.h

```diff
@@ -0,0 +1,59 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+static NSString *const SENTRY_IO_WRITE_OPERATION = @"file.write";
+
+static NSString *const SENTRY_IO_READ_OPERATION = @"file.read";
+
+@interface SentryNSDataTracker : NSObject
```

Comment:
`h`: Can we reuse this class when we also want to track file IO with FileManager? Do we need to change a lot in here?

Reply:
FileManager uses NSData to read and write files. 
Im creating tests to keep track of this behavior in future iOS SDK versions.